pages/index.js

```diff
@@ -1,12 +1,14 @@
 import Header from "@/components/Header/Header";
 import TaskList from "@/components/TaskList/TaskList";
+import CreateTaskForm from "@/components/CreateTaskForm/CreateTaskForm";
 
-export default function HomePage({ sortedDefaultTasks }) {
+export default function HomePage({ currentTasks, onCreateTask }) {
   return (
```

Comment:
Complementing my other suggestions, you should move the state here and write a helper function that gets called when the cancel button gets clicked:



```suggestion
export default function HomePage({ currentTasks, onCreateTask }) {
const [createMode, setCreateMode] = useState(false);

  function handleCancel() {
    setCreateMode(false);
  }
  return (
```